### PR TITLE
Kinetic panda moveit commander

### PR DESCRIPTION
**Pull request by mlautman**
[Live version](http://tutorials.picknik.ai/doc/moveit_commander/moveit_commander_tutorial.html)

### CODE REVIEW

File: doc/moveit_commander/moveit_commander_tutorial.rst [hunk 1]
```diff
@@ -1,41 +1,44 @@
 MoveIt! Commander and Command Line Tool
 =======================================
 
-The `moveit_commander <http://wiki.ros.org/moveit_commander>`_ Python package offers wrappers for the functionality provided in MoveIt!. Simple interfaces are available for motion planning, computation of Cartesian paths, and pick and place.
+The `moveit_commander <http://wiki.ros.org/moveit_commander>`_ Python package offers wrappers for the functionality provided in MoveIt!. Simple interfaces are available for motion planning, computation of Cartesian paths, and pick and place. The **moveit_commander** package also includes a command line tool for controlling the robot.
```
Comment (davetcoleman) [inline doc/moveit_commander/moveit_commander_tutorial.rst]: tool -> interface

"controlling the robot interactively" ... when i first read this this sentence seemed to repeat the rest of the paragraph

File: doc/moveit_commander/moveit_commander_tutorial.rst [hunk 2]
```diff
@@ -1,41 +1,44 @@
...
+  source ~/ws_moveit/devel/setup.bash
 
-First, launch the MoveIt! simulated environment interface: ::
+Start Rviz and wait for everything to finish loading in the first shell::
 
- roslaunch pr2_moveit_config demo.launch
+  roslaunch panda_moveit_config demo.launch
 
-Now initiate the **moveit_commander** interface: ::
+Now initiate the ``moveit_commander`` interface in another shell: ::
```
Comment (davetcoleman) [inline doc/moveit_commander/moveit_commander_tutorial.rst]: in the intro paragraph you make package names bold, here you make them in code format. please be consistent throughout the tutorials

File: doc/moveit_commander/moveit_commander_tutorial.rst [hunk 3]
```diff
@@ -1,41 +1,44 @@
...
+This command, ``current``, will show you the current state of your group: ::
 
  current
 
 To record that state under a specific name you can simply type: ::
 
  rec c
 
-This will remember the current joint values of the robot group under the name c. Matlab-like syntax is available for modifying joint values. The code above copies the joint values of c into a new variable named goal. We then modify the first joint of goal to 0.2. You may need to use a different value instead of 0.2 (it needs to be within your allowed bounds and not cause collisions). The **go** command plans a motion and executes it.
+This will remember the current joint values of the robot group under the name ``c``. Matlab-like syntax is available for modifying joint values. The code above copies the joint values of c into a new variable named goal. We then modify the first joint of ``goal`` to ``0.2``. You may need to use a different value instead of ``0.2`` (it needs to be within your allowed bounds and not cause collisions). The ``go`` command plans a motion and executes it.
```
Comment (davetcoleman) [inline doc/moveit_commander/moveit_commander_tutorial.rst]: great formatting improvements!

**Comment by mlautman** [comment]
@davetcoleman ready for merge?

**Pull request merged**